### DISCUSSION TITLE
空白ページが自動的に表示される

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1854] [Q&A] [answered]
序文 3 に別のページを追加しようとしましたが、後で空白のページが表示されました。序文 4 ページを追加すると、空白のページが削除されました。しかし、別の序文 5 を追加すると、再び空白のページが表示されます。私はすでにコマンド\let\cleardoublepage\clearpageを使用して空白ページを削除していますが、機能していません。

当面の間、TOCも無効にします。

序文の1ページ目にページカウンターをセットしました。これは \setcounter{page}{1} のせいですか?

空白ページが自動的に表示される。
(review-custom.sty)
`\let\cleardoublepage\clearpage`

(preface.re)
`#@# Comment
//embed[latex]{
\setcounter{page}{1}
  preface 1
\clearpage
  Preface 2
\clearpage
  Preface 3
//}
#@# Comment`

**Comment by kmuto** [comment, marked as answer]
あなたは本物の本を手元で開いて、その作り方を学ぶ必要があります。

- 一般的な横書きの本は、左ページが偶数even、右ページが奇数oddです。
- PREDEF (frontmatter部) にi〜からのroman number、CHAPS (mainmatter部) に 1〜 からのarabic number をページカウンターとする場合、arabic number 1にスイッチするため、必ず奇数・右ページになる必要があります。よって、frontmatter部が奇数で終わってしまった場合は、偶数の左ページとして、空白ページが必ず入ります。
- また、デフォルトでは章(=)は必ず奇数で始まります(あなたはletで無理矢理変えているようですが、これは推奨される方法ではありません)。

どうしたらよいか？

- まず、config.ymlのtexdocumentclassの2つめのパラメータに、「,openany,serial_pagination=true」を加えます。openanyは章の始まりを偶数でも許容するものです(これにより、あなたのreview-customにあるlet行は不要です)。serial_paginationはRe:VIEW固有の拡張ですが、PREDEFのページカウンターをarabicにし、CHAPSも番号を継続します。これにより、roman/arabicの切り替えがなくなる、つまり、CHAPSで1にリセットされることがなくなります。
- さらに、jsbookの場合、デフォルトではmainmatterへの切り替え時に奇数にすることが固定されているので、これを上書きして変更します。review-custom.styに以下のように記述します。
```
\renewcommand\mainmatter{
  \@mainmattertrue
}
```

Re:VIEWは本物の本を作るためのツールであり、電子的なドキュメントを作るのは二次的です。たとえば周囲の余白や空白ページにはすべて理由があります。
